src/main/java/io/github/intoto/slsa/models/v02/Provenance.java

```diff
@@ -109,6 +109,6 @@ public int hashCode() {
 
   @Override
   public String getPredicateType() {
-    return "https://slsa.dev/provenance/v0.1";
+    return "https://slsa.dev/provenance/v0.2";
```

Comment:
v1?

Reply:
no I think this is ok. It's a fix on a previous issue

Reply:
my bad!